lib/use-checkout-pricing.js

```diff
@@ -38,20 +47,39 @@ export default function useCheckoutPricing(initialInputs = {}, handleError = thr
   const recurly = useRecurly();
   const [loading, setLoading] = useState(false);
   const [input, setInput] = useState(initialInputs);
-  let checkoutPricing = recurly.Pricing.Checkout();
-  const [pricing, setPricing] = useState(checkoutPricing);
+  const [pricing, setPricing] = useState(recurly.Pricing.Checkout());
 
   useEffect(() => {
     setLoading(true);
-    const { subscriptions = [] } = input;
+    let checkoutPricing = recurly.Pricing.Checkout();
```

Comment:
Why did this need to move into the effect?

Reply:
This was done in to this warning 
![image](https://user-images.githubusercontent.com/32269552/73560034-d6fe0980-441b-11ea-81ce-e879dc8e2469.png)

Reply:
We can also follow the warning's suggestion to store the value in a useRef hook instead if that's more appropriate

Reply:
Starting fresh in the effect seems perfectly reasonable to me. The request layer will cache as needed, so the perf impact should be small.